docs/ATLAS.md

```diff
@@ -0,0 +1,23 @@
+### Setup Atlas using terraform ###
+
+Follow these steps to setup a M2 cluster in Atlas to be used by the uDeploy portal.
```

Comment:
Maybe remove this line completely since udeploy is implied by context, and Atlas is in the header above and M2 is defined in the terraform scripts. Less Reading Simpler Process (LRSP)